pyEpiabm/pyEpiabm/property/place_foi.py

```diff
@@ -38,7 +38,10 @@ def place_inf(place, infector, time: float):
         params = Parameters.instance().place_params
         transmission = params["place_transmission"]
         place_idx = place.place_type.value - 1
-        num_groups = params["mean_group_size"][place_idx]
+        try:
+            num_groups = params["mean_group_size"][place_idx]
+        except IndexError:  # For place types not in parameters
+            num_groups = 1
```

Comment:
I am however wondering why do you take this value for ``num_groups``. Could you please explain the reasoning?

Reply:
@Elizabeth-Hayman  has only specified the group sizes for some place types in the parameters file - I believe this is because other place types don't have separate groups of people. However this was not accounted for in the `place_foi`, causing an error. This just means that if a place does not have separate group sizes specified, we will assume the place only have one group (for which the mean place capacity is used)